python/ray/serve/_private/proxy_state.py

```diff
@@ -318,15 +319,19 @@ def __init__(
         actor_name: str,
         node_id: str,
         node_ip: str,
+        proxy_restart_count: int = 0,
+        time_getter: ModuleType = time,
```

Comment:
what is the `time` (default value)?

Reply:
This is the default `time` module in python

Reply:
Was written this way so we can inject a fake object to control the time during the test

Reply:
I can't find you define the `time` variable somewhere...

Reply:
do we really need to have default value for these two arguement?

Reply:
It's in top of the file `import time`😅

I tend to add defaults for any new variables I added to ensure things are backward compatible and has a default behavior even if the caller doesn't do anything. In this case `time_getter` as an injection should definitely have a default. I think we can go without default for `proxy_restart_count`, but would expand the changes quite a lot.